### PR TITLE
fix: duplicate live feed links

### LINKED ISSUE #1914: Duplicate links in Live Feed list-items
**Actual Behaviour**

Some of the list-items in Live Feed have links in the message as well as a separate TextView with link. 
So as a result the link is duplicated.

**Expected Behaviour**

One of the links should be removed

**Screenshots of the issue**

screenshot_2017-09-15-02-31-33-318_org fossasia openevent

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by manav-mj**
Fixes #1914 

Changes: 
Removes unclickable duplicate link in live feed

Screenshot of the issue:

Screenshots for the change:

### CODE REVIEW

File: android/app/src/main/java/org/fossasia/openevent/adapters/FeedAdapter.java [hunk 1]
```diff
@@ -107,7 +107,10 @@ public void onBindViewHolder(RecyclerViewHolder holder, int position) {
         }
 
         if (!TextUtils.isEmpty(feedItem.getMessage())) {
-            holder.statusMsg.setText(feedItem.getMessage());
+            String statusMsg = feedItem.getMessage();
+            if (feedItem.getLink() != null)
+                statusMsg = statusMsg.replace(feedItem.getLink(), "");
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/adapters/FeedAdapter.java]: Can you please add a comment regarding the reason for replacing the string for future developers? :wink:

File: android/app/src/main/java/org/fossasia/openevent/adapters/FeedAdapter.java [hunk 2]
```diff
@@ -63,10 +63,10 @@
             getComments.setOnClickListener(v -> {
                 FeedItem clickedFeedItem = feedItems.get(getPosition());
                 commentItems = new ArrayList<>();
-                if(clickedFeedItem.getComments() != null) {
+                if (clickedFeedItem.getComments() != null) {
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/adapters/FeedAdapter.java:66]: While the formatting change you did was correct, but it is generally considered good practice to only change parts of code related to the issue you are solving. This makes it easier for reviewing and track changes.

If you want to fix the formatting, you can create another issue just for it and fix the formatting of entire project in a new PR :smile:

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

File: android/app/src/main/java/org/fossasia/openevent/adapters/FeedAdapter.java (hunk 2 above)
Comment (manav-mj) [reply android/app/src/main/java/org/fossasia/openevent/adapters/FeedAdapter.java:66]: @iamareebjamal 
i'll keep that in mind in the future. Actually i used a shortcut to change the formatting so it automatically changed the other code and it went unnoticed.
Should i change the formatting back to original?

Comment (iamareebjamal) [reply android/app/src/main/java/org/fossasia/openevent/adapters/FeedAdapter.java:66]: No leave it be for this PR

**Review by scandindian — APPROVED** [review]

**Comment by manav-mj** [comment]
@iamareebjamal Pls. review again.

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**